Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 platforms :rbx do
-  gem 'rubysl', '~> 2.0'
+  #gem 'rubysl'
```

Comment:
is this gem not needed anymore? or what is the reason the you commented it out?

Reply:
It's incompatible with ruby 3.1

Reply:
I see that this gem has not been maintained since 2016. Could you please remove it when it is no longer needed?

Reply:
I'd like to keep it just for future reference. It was there for a reason. Maybe in the future we'll have to find some replacement for it.